data/korath/korath outfits.txt

```diff
@@ -652,7 +652,141 @@ outfit "Steering (Stellar Class)"
+	description "	Despite the Efreti’s limited resources, these new hybrid gravitoplasma thrusters allow their engines to maintain the traditional Korath emphasis on speed in a small package while keeping heat output down."
+
+outfit "Nelmeb GP Hybrid Thruster"
+	plural "Nelmeb GP Hybrid Thrusters"
+	category "Engines"
+	"cost" 1649000
+	thumbnail "outfit/large efreti thruster"
+	"mass" 86
+	"outfit space" -86
+	"engine capacity" -86
+	"thrust" 105
+	"thrusting energy" 14
+	"thrusting heat" 18
+	"flare sprite" "effect/efreti flare/large"
+		"frame rate" 6.5
+	"flare sound" "ion large"
+	description "Korath physicists have, for several centuries, understood powerful aspects of the unity that underlies both quantum gravity and forces that bind quarks and gluons together. Recently, prompted by the example of the Quarg's graviton thrusters, Efreti engineers have begun to utilize their theoretical knowledge of this unity to incorporate gravitational propulsion into their more traditional fusion-based plasma engines. However, as Efreti efforts are comparatively recent, this technological synthesis lags behind comparable Quarg technology."
+	description "	These are the largest such gravitoplasma-thrusters that the Efreti have to offer, and as such are usually only found on their gargantuan civilian ships. Despite their size and thrusting power, the Quarg are rather unimpressed with the inefficient design of such engines."
+
+outfit "Arkrof GP Hybrid Steering"
+	plural "Arkrof GP Hybrid Steerings"
+	category "Engines"
+	"cost" 131000
+	thumbnail "outfit/tiny efreti steering"
+	"mass" 12
+	"outfit space" -12
+	"engine capacity" -12
+	"turn" 323
+	"turning energy" 1.2
+	"turning heat" 1.225
+	"steering flare sprite" "effect/efreti flare/tiny"
+		"frame rate" 8
+	"steering flare sound" "ion tiny"
+	description "Unlike most Korath engines, Efreti engines do not rely solely on plasma. Gravitoplasma engines combine traditional Korath plasma thrust with Quarg-inspired forays into gravitation-based propulsion. The smallest steering manages to develop a great deal of torque at the expense of higher energy costs."
+
+outfit "Farves GP Hybrid Steering"
+	plural "Farves GP Hybrid Steerings"
+	category "Engines"
+	"cost" 413000
+	thumbnail "outfit/small efreti steering"
+	"mass" 24
+	"outfit space" -24
+	"engine capacity" -24
+	"turn" 737
+	"turning energy" 2.5
+	"turning heat" 2.75
+	"steering flare sprite" "effect/efreti flare/small"
+		"frame rate" 7.5
+	"steering flare sound" "ion small"
+	description "Exposure to the Quarg's graviton-thrusters, coupled with the forced retreat from research into interstellar Jump-Drive technology, has left the Efreti engineers with little else to tinker with than subluminal-propulsion. Their ongoing efforts represent their evolving understanding of fundamental physics and how it can be put into practice."
```

Comment:
```suggestion
	description "Exposure to the Quarg's graviton-thrusters, coupled with the forced retreat from research into interstellar jump drive technology, has left the Efreti engineers with little else to tinker with than subluminal propulsion. Their ongoing efforts represent their evolving understanding of fundamental physics and how it can be put into practice."
```